### ISSUE TITLE
Manage routing.

### ISSUE THREAD

**Issue by nabily4e-dev** [202212-GIZ-YE-FEW/react-ecommerce-reactifyecom#10]
* Clicking on a category, should take user to the single product page of this category.
* Clicking on a product among list of products, should take user to that single product page.
* Clicking on a product among results of products, should take user to that single product page.
* Links navbar pages to corresponding pages.
* others...

**Comment by sulmi24** [comment]
Do we need to make a custom routing? I believe If you have created a file in the pages directory of your Next.js application, **Next.js will automatically generate a route for that file** based on the file name and location in the directory structure.

**Comment by nabilashbat** [comment]
Yeah, I mean it needs a specific or a sort of dedicated structure so this task only there for someone already knows how to structure the routing methodology followed or Built-in in Next.js.  So this is basically like a search task, you try to know to implement it in the app.

**Comment by sulmi24** [comment]
I will try to fix them and update you guys
